src/Extension/ExtendableTrait.php

```diff
@@ -410,16 +410,16 @@ public function extendableCall($name, $params = null)
             $extension = $this->extensionData['methods'][$name];
             $extensionObject = $this->extensionData['extensions'][$extension];
 
-            if (method_exists($extension, $name) && is_callable([$extension, $name])) {
-                return call_user_func_array([$extensionObject, $name], $params);
+            if (method_exists($extension, $name)) {
+                return call_user_func_array([$extensionObject, $name], array_values($params));
```

Comment:
Reason for change?

Reply:
`is_callable()` returns `false` on callable that calls non-static methods statically. It returned true prior to PHP 8.0. `is_callable()` will fail when checking for a non-static method with a classname (must check with an object instance)